khmer/khmer_args.py

```diff
@@ -46,15 +43,61 @@ def build_hash_args(descr=None, epilog=None):
     parser.add_argument('--ksize', '-k', type=int, default=env_ksize,
                         help='k-mer size to use')
     parser.add_argument('--n_tables', '-N', type=int,
-                        default=env_n_tables,
+                        default=None,
```

Comment:
This is effectively changing the command line API. Now a test is failing because we call 'normalize-by-median' with a `--min-tablesize` but without `--n_tables` as it relied on the default value.